RELEASE.md

```diff
@@ -1,3 +1,23 @@
+# Release 0.0.21
+
+## New features
+- Load custom SBERT models from cloud storage with authentication (https://github.com/marqo-ai/marqo/pull/474). 
+Marqo now supports fetching your fine-tuned public and private SBERT models from Hugging Face and AWS s3. For usage please refer to our [documentation](https://docs.marqo.ai/0.0.21/Models-Reference/bring_your_own_model/#bring-your-own-hugging-face-sbert-models). For instructions on loading a private model using authentication, check
```

Comment:
Wording:

`Face and AWS s3. For usage please refer to our [documentation]... `
-> 
`Face and AWS s3. Learn more about using your own SBERT model [here]... `